src/main/java/io/cryostat/net/PermissionDeniedException.java

```diff
@@ -24,7 +24,7 @@ public PermissionDeniedException(
             String namespace, String resource, String verb, String reason) {
         super(
                 String.format(
-                        "Requesting client in namespace \"%s\" cannot %s %s: %s",
+                        "Requested client in namespace \"%s\" cannot %s %s: %s",
```

Comment:
I think this one actually should be "Requesting", or maybe another word entirely. What this is logging is that some API client (user) asked to perform some action, and Cryostat determined that this user does not have sufficient permissions to do what they had asked.

Reply:
I'll close this PR for now then.